code/modules/cargo/bounties/chef.dm

```diff
@@ -263,11 +270,30 @@
 	reward_high = 5700
 	required_count = 3
 	random_count = 1
-	wanted_types = list(/obj/item/reagent_containers/food/snacks/lortl,
-			/obj/item/reagent_containers/food/snacks/soup/qilvo,
-			/obj/item/reagent_containers/food/snacks/soup/zantiri,
-			/obj/item/reagent_containers/food/snacks/xuqqil,
-			/obj/item/reagent_containers/food/snacks/qlguabi)
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/)
+
+/datum/bounty/item/chef/skrell/applies_to(var/obj/item/reagent_containers/food/snacks/O)
+	if(!..())
+		return FALSE
+	if(O.icon == 'icons/obj/item/reagent_containers/food/cultural/skrell.dmi')
+		return TRUE
```

Comment:
checking the icon isn't an acceptable way to do this - icon files are prone to being changed and generally should never be used to check the state or type of an object